playgrounds/vite/src/app.tsx

```diff
@@ -2,7 +2,7 @@ import { Foo } from './foo'
 
 export function App() {
   return (
-    <div className="m-3 p-3 border">
+    <div className="m-3 p-3 border shadow-xl shadow-red-500">
```

Comment:
Not sure if you wanted to commit this part. But other than that, looking good!

Reply:
Jesus christ how many things can I fuck up in the smallest diff possible?